Codeinwp/neve
# Feat: Custom Post Type Enhancements

### Summary
Custom Post Type Enhancement feature changes.
 - Allows changing Sidebar options for custom post types in Customizer
 - Allows changing Container options for custom post types in Customizer
 - Single Custom Post Type options available, by default it inherits the Single Post settings
 - Archive Custom Post Type options if archive is enabled for custom post type, by default it inherits from Blog settings.

### Will affect visual aspect of the product
NO


### Test instructions
1. Activate the Post Enhancement Feature, to make sure no confusion is made about what should be available, disable the Blog Pro Module
2. Check that Settings are being respected from Customizer for the Custom Post Types, also for the Archive pages.

Closes  Codeinwp/neve-pro-addon#1767.
